src/pages/feed.js

```diff
@@ -42,17 +42,15 @@ export default class Feed extends Component {
       .post(process.env.REACT_APP_FEED, { counter: this.state.counter + 1 })
       .then((res) => {
         const filtered = res.data.result.filter((e) => {
-          if (
+          return (
             e.token_id !== 1130 &&
             e.token_id !== 1131 &&
             e.token_id !== 1417 &&
             e.token_id !== 1418 &&
             e.token_id !== 1419 &&
             e.token_id !== 641 &&
             e.token_id !== 1547
-          ) {
-            return e
-          }
+          )
```

Comment:
same thing but always returns `true` or `false` instead of `true` or `undefined`